app/src/vanilla/java/cm/aptoide/pt/search/view/SearchResultPresenter.java

```diff
@@ -306,21 +307,24 @@ private void openAppView(SearchAppResultWrapper searchApp) {
     // search every store. followed and not followed
     return Single.zip(loadDataFromFollowedStores(query, onlyTrustedApps, 0),
         loadDataFromNonFollowedStores(query, onlyTrustedApps, 0),
-        (followedStoresCount, nonFollowedStoresCount) -> {
-          int result = 0;
-          if (followedStoresCount != null && followedStoresCount.size() > 0) {
-            result += followedStoresCount.size();
-          } else {
-            view.hideFollowedStoresTab();
+        new Func2<List<SearchAppResult>, List<SearchAppResult>, Integer>() {
```

Comment:
insert lambda ( I think this was when we were debugging the return type issue in another chain )